src/component/header.js

```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+import Styled from 'styled-components';
+import FontAwesome from 'react-fontawesome';
+
+import Themeline from './shared/themeline';
```

Comment:
First imports should be package imports.
Add a space and...
Component imports
Add a space and...
Image file imports
Add **two** spaces and continue...